project/profiles/capacity4more/modules/c4m/user/c4m_user_profile/c4m_user_profile.module

```diff
@@ -298,6 +298,21 @@ function c4m_user_profile_user_login(&$edit, $user) {
 }
 
 /**
+ * Implements hook_mail_alter().
+ *
+ * For users that requested email notifications format to be plaintext,
+ * sets 'email confirmation' mail format accordingly.
+ */
+function c4m_user_profile_mail_alter(&$message) {
+  if ($message['id'] != 'email_confirm_confirmation_email') {
```

Comment:
@anvmn 
this should work for all emails that the user receive from the system, not only `email conform confirmation email`